src/test/java/org/medicmobile/webapp/mobile/SmsSenderTest.java

```diff
@@ -46,6 +46,7 @@
 import java.util.stream.IntStream;
 
 @RunWith(RobolectricTestRunner.class)
+@Config(sdk = 31) // ToDo: Remove when upgrading robolectric
```

Comment:
We're getting this error because our current version of `robolectric` doesn't support `target 33`

```
java.lang.IllegalArgumentException at RobolectricTestRunner.java:260
        Caused by: java.lang.IllegalArgumentException at DefaultSdkPicker.java:118
```
In this [link](https://android.googlesource.com/platform/frameworks/support/+/androidx-main/docs/testing.md#robolectric), they recommend to use `Config(sdk = 31)` to make the test pass as a _temporary fix_.